### PR TITLE
Wallet modify

### PR DESCRIPTION
**Pull request by ckdbswo**

### CODE REVIEW

File: src/containers/walletContainer.js [hunk 1]
```diff
@@ -123,50 +123,45 @@ class WalletContainer extends Component {
...
-            open: false
-        })
     }
 
 
     renderEditable(cellInfo, flag) {
         if (flag === "deposit") {
             return<div>
-                <button type="button" onClick={(e)=>this.onOpenModalA(e, cellInfo)} value="deposit">Deposit</button>
+                <button type="button" onClick={(e)=>this.onOpenModalA(e, cellInfo)} value="deposit">Deposit_pop</button>
```
Comment (aman-mahalgavan) [inline src/containers/walletContainer.js]: change to Deposit

File: src/containers/walletContainer.js [hunk 2]
```diff
@@ -123,50 +123,45 @@ class WalletContainer extends Component {
...
             </div>
         } else {
             return <div>
-                <button type="button" onClick={(e)=>this.onOpenModalA(e, cellInfo)} value="withdraw">Withdraw</button>
+                <button type="button" onClick={(e)=>this.onOpenModalA(e, cellInfo)} value="withdraw">Withdraw_pop</button>
```
Comment (aman-mahalgavan) [inline src/containers/walletContainer.js]: change to withdraw

File: src/containers/walletContainer.js (hunk 1 above)
Comment (ckdbswo) [reply src/containers/walletContainer.js]: changed button name

File: src/containers/walletContainer.js (hunk 2 above)
Comment (ckdbswo) [reply src/containers/walletContainer.js]: changed button name

**Pull request merged**